python/pyiceberg/cli/console.py

```diff
@@ -103,7 +103,7 @@ def list(ctx: Context, parent: Optional[str]):  # pylint: disable=redefined-buil
 @click.pass_context
 @catch_exception()
 def describe(ctx: Context, entity: Literal["name", "namespace", "table"], identifier: str):
-    """Describes a namespace xor table"""
```

Comment:
This is actually a bad joke. If you have hierarchical namespaces, then it will describe the namespace, `namespace.namespace.table`, in the case of a table `namespace.table` it would describe the table. But wouldn't do both (because that's not possible). But I agree that this is better :)